home01/js/script_8.js

```diff
@@ -0,0 +1,5 @@
+// 8. Пользователь вводит сумму денег в кошельке и цену одной шоколадки. Программа выводит сколько шоколадок может купить пользователь и сколько сдачи у него останется.
+const sumManey = +prompt(`Enter the amount of money in the wallet:`);
+const numChokolat = +prompt(`Enter number of chocolates:`);
```

Comment:
пользователь должен ввести цену
```suggestion
const chocolatePrice = +prompt(`Enter price of one chocolate bar:`);
```